web/console/src/app/components/common/Registration/index.tsx

```diff
@@ -12,7 +12,7 @@ import StorageHandler from '@/velas/storageHandler';
 // @ts-ignore
```

Comment:
why ts-ignore?

Reply:
because it is js file, not ts. This file was represented in velas docs